NuRadioReco/modules/phasedarray/phasedArray.py

```diff
@@ -0,0 +1,215 @@
+from NuRadioReco.modules.base.module import register_run
+from NuRadioReco.utilities import units
+import logging
+import scipy
+import numpy as np
+from scipy import constants
+
+logger = logging.getLogger('phasedTriggerBase')
+
+cspeed = constants.c * units.m / units.s
+
+class phasedArray():
```

Comment:
Is it worth to change the class name to something like `phasedArrayTrigger` or `phasedArrayTriggerBase`?